.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5502
```

Comment:
You can have the port changes specific to your computer on your local repo, but wouldn't recommend changing them here in case they clash with documentation.

Reply:
I have deleted this part